src/extensionsIntegrated/CSSColorPreview/main.js

```diff
@@ -142,6 +144,45 @@ define(function (require, exports, module) {
         });
     }
 
+
+    /**
+     * To move the cursor to the color text and display the color quick edit
+     * @param {Editor} codem the codemirror instance
+     * @param {Number} lineNumber the line number that is clicked
+     * @param {String} gutter the gutter name
+     */
+    function colorIconClicked(codem, lineNumber, gutter) {
+        const editor = EditorManager.getActiveEditor();
+        const cm = editor._codeMirror;
+
+        if(gutter === 'CodeMirror-linenumbers') {
+
+            let colorValue;
+
+            for(let i of codem.colorGutters) {
+                if(i.lineNumber === lineNumber) {
+                    colorValue = i.colorValues[0];
+                }
+            }
+
+            const lineText = cm.getLine(lineNumber);
```

Comment:
Use editor.getline api. Avoid using cm object directly whenever possible as we will be moving to new version of codemirror 6 which haave breaking api changes

Reply:
The API didn't exit so added in https://github.com/phcode-dev/phoenix/pull/1965 , check the file editor.js